src/main/java/dev/masa/masuitechat/bungee/channels/Private.java

```diff
@@ -67,24 +70,20 @@ public void create(ProxiedPlayer sender, ProxiedPlayer receiver, String msg) {
                 .replace("%receiver_prefix%", receiverInfo.getPrefix())
                 .replace("%receiver_suffix%", receiverInfo.getSuffix())
         );
-        if (sender.hasPermission("masuitechat.chat.colors")) {
```

Comment:
Oh yeah, I also cleaned up this code a bit. Basically didn't need this if-statement because it was already stated below.